src/Dockerfile

```diff
@@ -30,7 +29,7 @@ FROM amsterdam/python:3.7-slim-buster
 COPY --from=builder /usr/local/bin/ /usr/local/bin/
 COPY --from=builder /usr/local/lib/ /usr/local/lib/
 COPY --from=builder /usr/lib/x86_64-linux-gnu/libaio* /usr/lib/x86_64-linux-gnu/
-COPY --from=builder /usr/local/lib/python3.7/site-packages/ /usr/local/lib/python3.7/site-packages/
+COPY --from=builder /usr/local/lib/python3.8/site-packages/ /usr/local/lib/python3.8/site-packages/
 COPY --from=builder /usr/bin/ /usr/bin/
 COPY --from=builder /usr/lib/x86_64-linux-gnu/libcurl* /usr/lib/x86_64-linux-gnu/
 COPY --from=builder /usr/lib/x86_64-linux-gnu/perl5/. /usr/lib/x86_64-linux-gnu/perl5/
```

Comment:
zijn al deze copies echt nodig? klinkt alsof er een paar packages missen in de runtime image.

Reply:
Dat kan @bheupers @jjmurre en @cvriel beter beantwoorden. Ik durf het niet weggoien, omdat er zijn geen tests voor dit deel.

Reply:
eens, dat mag een aparte opschoonactie zijn wat mij betreft!